readme/Readme.scalatex

```diff
@@ -540,6 +540,45 @@
     @p
       The rest of the code involved in this micro-benchmark can be found in @lnk("PerfTests.scala", "shared/test/scala/scalatags/PerfTests.scala")
 
+   @sect{Utilities}
+
+    @p
+      There's a simple converter to produce simply indented text with one fragment per line.  This allows you to inspect the html output.  Using scala.xml to prettyprint the output would somtimes render the html invalid.
+    @hl.highlight("""
+    scala> import scalatags.Text.all._
+    import scalatags.Text.all._
+
+    scala> import scalatags.text.Utils.Pretty
+    import scalatags.text.Utils.Pretty
+
+    scala> div(
+         |   p(float.left)(
+         |     "This is my first paragraph"
+         |   ),
+         |
+         |   a(tabindex:=10)(
+         |     p("Goooogle")
+         |   ),
+         |   input(disabled:=true)
+         | ).pretty
+    res6: scalatags.text.Utils.Pretty =
+    <div>
+      <p style="float: left;">
+        This is my first paragraph
+      </p>
+
```

Comment:
@lihaoyi do you have a strong preference re combining tags on one line like this:
`        <p> Goooogle </p>`

or do you think this is better:

```
        <p>
            Goooogle
        </p>
```

My feeling is to go with the second.